README.md

```diff
@@ -220,3 +224,34 @@ This may be better suited for you if you plan to change some of the automaticall
 ## Recap
 
 After following this complete README, we should have an ETH smart contract as well as a Starknet smart contract, both connected to act as a bridge between these two chains.
+
+## Upgrade Contracts in Testnet
+
+### Starknet
+
+If you want to upgrade a previously deployed `Escrow` contract, it is possible through a command. We will perform the upgrade using `starkli`, so the same configuration used for deployment is necessary.
+
+Keep in mind that this command will **rebuild** `Escrow.cairo`, perform the **declare**, and call the external **upgrade()** function with the new class hash resulting from the **declare**. 
+
+To be able to upgrade the contract, you must be the **owner** of the contract and set the `ESCROW_CONTRACT_ADDRESS` variable in the `.env` file with the address of the `Escrow` contract that you want to be **upgraded**.
+
+1. Update `contracts/cairo/.env` file.
+
+```bash
+   ..
+   ## Starkli
+   STARKNET_ACCOUNT = Absolute path of your starknet testnet account, created at the start of this README
+   STARKNET_KEYSTORE = Absolute path of your starknet testnet keystore, created at the start of this README
+   SN_RPC_URL = Infura or Alchemy RPC URL
+   
+   ..
+   # To upgrade Starknet contract 
+   ESCROW_CONTRACT_ADDRESS = Address of the Starknet smart contract, this value should be empty, and is automatically 
```

Comment:
This value should not be empty. The variable should be in the .env during deploy (either empty or with a value, it will be overrwritten).
Then, in upgrade, this value should definitely be set to the smart contract that wants to be upgraded.

Reply:
youre right, i change this message to 
```
# To upgrade Starknet contract 
ESCROW_CONTRACT_ADDRESS = You can either set an escrow address manually or run deploy.sh, and it will be set automatically
```
i think its better, what do you think?